squarify/__init__.py

```diff
@@ -254,14 +278,14 @@ def plot(
         x, dy, width=dx, bottom=y, color=color, label=label, align="edge", **bar_kwargs
     )
 
-    if value is not None:
+    if not value is None:
```

Comment:
I think the original reads more clearly. Python allows this specifically for this reason.